docs/04-testing.md

```diff
@@ -31,25 +31,17 @@ Then, copy the following template to use an in-memory sqlite database and enable
 
 ```xml title="phpunit.xml"
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  bootstrap="vendor/autoload.php"
-  backupGlobals="false"
-  backupStaticAttributes="false"
-  colors="true"
-  verbose="true"
-  convertErrorsToExceptions="true"
-  convertNoticesToExceptions="true"
-  convertWarningsToExceptions="true"
-  processIsolation="false"
-  stopOnFailure="false"
-  xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+<phpunit 
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+     bootstrap="vendor/autoload.php" 
+     backupGlobals="false" colors="true" 
+     processIsolation="false" 
+     stopOnFailure="false" 
+     xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd" 
+     cacheDirectory=".phpunit.cache" 
```

Comment:
Pls remove this as .phpunit.cache is not a directory

Reply:
Actually  `.phpunit.cache` is a directory in my machine. I think it contains the last executed tests result and code coverage reports.

Reply:
oh weird